### PR TITLE
HTTP sources fail to sync (HTTP GET Request) #2910

### LINKED ISSUE #2910: HTTP sources fail to sync (HTTP GET Request)
## Expected Behavior

Given open source data like this:

`https://chronicdata.cdc.gov/resource/g4ie-h725.json`

I would like to store this data in Postgres.

## Current Behavior

The source and destination configuration was successful.  I specified the HTTP source as shown above.  the sync was not successful.  The error was 

`Caused by: io.airbyte.workers.WorkerException: Tap process wasn't successful`

## Logs
See attached log

## Steps to Reproduce
1. Configure source and destination as specified
1. Run a manual sync.
1.  Sync fails

## Severity of the bug for you
Medium

## Airbyte Version

VERSION=0.20.0-alpha

## Connector Version (if applicable)
HTTP 0.2.2

## Additional context
Docker image, MacOS 10.14.6
[logs-2-0.txt](https://github.com/airbytehq/airbyte/files/6320821/logs-2-0.txt)

### PR DESCRIPTION
**Pull request by vovavovavovavova**
closes #2910 
## What
Supports array of Objects in source_http_request

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

File: airbyte-integrations/connectors/source-http-request/source_http_request/source.py [hunk 1]
```diff
@@ -71,14 +71,28 @@ def read(
...
         # need to eagerly fetch the json.
-        message = AirbyteMessage(
-            type=Type.RECORD,
-            record=AirbyteRecordMessage(
-                stream=SourceHttpRequest.STREAM_NAME, data=r.json(), emitted_at=int(datetime.now().timestamp()) * 1000
-            ),
-        )
-
-        return (m for m in [message])
+        data = r.json()
```
Comment (keu) [inline airbyte-integrations/connectors/source-http-request/source_http_request/source.py:74]: ok, lets make it DRY
```python
if not isinstance(data, list):
   data = [data]

for record in data:
    yield AirbyteMessage(
                    type=Type.RECORD,
                    record=AirbyteRecordMessage(
                        stream=SourceHttpRequest.STREAM_NAME,
                        data=obj,
                        emitted_at=int(datetime.now().timestamp()) * 1000
                    )
                )
```

**Review by keu — CHANGES_REQUESTED** [review]
suggestion

**Review by keu — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Pull request merged**